### ISSUE TITLE
If a scheduled job exists, and the backup target is changed, backups fail

### ISSUE THREAD

**Issue by tylarb** [yugabyte/yugabyte-db#6673]
Reproduction:
- Set up universe backup to NFS
- Set up a scheduled backup
- (backup succeeds)
- NOW, to to "configs" -> "backups" and change NFS storage directory

Scheduled backup now fails with this error:
```
  670 2020-12-15 02:06:13,002 [ERROR] from com.yugabyte.yw.commissioner.AbstractTaskBase in TaskPool-MultiTableBackup(f4122a4b      -f19b-466a-9861-b6a1fed6a90e)-0 - Errored out with: java.lang.NullPointerException
  671 2020-12-15 02:06:13,007 [ERROR] from com.yugabyte.yw.commissioner.SubTaskGroup in TaskPool-0 - Failed to execute task ja      va.util.concurrent.FutureTask@28e4d5e5, hit error java.lang.RuntimeException: java.lang.NullPointerException.
  672 java.util.concurrent.ExecutionException: java.lang.RuntimeException: java.lang.NullPointerException
  673 ⇥       at java.util.concurrent.FutureTask.report(FutureTask.java:122)
  674 ⇥       at java.util.concurrent.FutureTask.get(FutureTask.java:192)
  675 ⇥       at com.yugabyte.yw.commissioner.SubTaskGroup.waitFor(SubTaskGroup.java:175)
  676 ⇥       at com.yugabyte.yw.commissioner.SubTaskGroupQueue.run(SubTaskGroupQueue.java:42)
  677 ⇥       at com.yugabyte.yw.commissioner.tasks.MultiTableBackup.run(MultiTableBackup.java:237)
  678 ⇥       at com.yugabyte.yw.commissioner.TaskRunner.run(TaskRunner.java:143)
  679 ⇥       at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
  680 ⇥       at java.util.concurrent.FutureTask.run(FutureTask.java:266)
  681 ⇥       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  682 ⇥       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  683 ⇥       at java.lang.Thread.run(Thread.java:748)
  684 Caused by: java.lang.RuntimeException: java.lang.NullPointerException
  685 ⇥       at com.yugabyte.yw.commissioner.tasks.subtasks.BackupTable.run(BackupTable.java:88)
  686 ⇥       ... 5 common frames omitted
  687 Caused by: java.lang.NullPointerException: null
  688 ⇥       at com.yugabyte.yw.common.TableManager.addCommonCommandArgs(TableManager.java:221)
  689 ⇥       at com.yugabyte.yw.common.TableManager.runCommand(TableManager.java:139)
  690 ⇥       at com.yugabyte.yw.common.TableManager.createBackup(TableManager.java:242)
  691 ⇥       at com.yugabyte.yw.commissioner.tasks.subtasks.BackupTable.run(BackupTable.java:70)
  692 ⇥       ... 5 common frames omitted
  693 2020-12-15 02:06:13,009 [ERROR] from com.yugabyte.yw.commissioner.SubTaskGroupQueue in TaskPool-0 - SubTaskGroup 'Backup      Table : completed 0 out of 1 tasks.' waitFor() returned failed status.
```

**Comment by streddy-yb** [comment]
hi @Arnav15 - can you summarize the plan we discussed?

**Comment by chirag-yb** [comment]
It will be solved with this issue - https://github.com/yugabyte/yugabyte-db/issues/7349

**Comment by nishantSharma459** [comment]
Hi @chirag-yb I believe if the storage config is **'in use'** then we won't allow the user to update the storage path.
